Author: Robert James Clay <jame@rocasa.us>
Description: Remove pointless translation (fixes setup.pl listing the version incorrectly).
 Reference upstream: https://github.com/ledgersmb/LedgerSMB/commit/0673ffd431813a386559ec2e3810ce70257894cf
Origin: vendor
Last-Update: 2017-09-18
--- a/UI/login.html
+++ b/UI/login.html
@@ -45,7 +45,7 @@
             </div>
           <div style="z-index:10; position:absolute; top:0; left:0; width:100%; height:100%;">
           <h1 class="login" align="center">
-            <?lsmb text("LedgerSMB [_1]", VERSION) ?>
+            LedgerSMB <?lsmb VERSION ?>
           </h1>
           <div align="center">
             <div id="company_div"
--- a/UI/setup/credentials.html
+++ b/UI/setup/credentials.html
@@ -31,7 +31,7 @@
         alt="LedgerSMB Logo" />
       </a>
       <h2 align="center" style="margin-top: 0">
-        <?lsmb text("LedgerSMB [_1]", VERSION) ?>
+        LedgerSMB <?lsmb VERSION ?>
       </h2>
       <div class="listtop">
         <?lsmb text('Database administrator credentials') ?>
